### PR TITLE
nodejs-lts: update to 10.15.0.

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: srcpkgs/nodejs-lts/template [hunk 1]
```diff
@@ -1,32 +1,45 @@
...
 conflicts="nodejs"
 provides="nodejs-runtime-0_1"
 
 case "$XBPS_TARGET_MACHINE" in
-        ppc64le*) ;;
-        ppc64*) broken="Node is not supported on ppc64 BE Linux";;
+	arm*)
+		case "$XBPS_MACHINE" in
+			x86_64*|aarch64*)
+				nocross="Can't cross-compile to 32bit-host from 64bit-host";;
```
Comment (maxice8) [inline srcpkgs/nodejs-lts/template]: 32bit-target

**Comment by maxice8** [comment]
@the-maldridge 

Update of LTS to new major version so i'm tagging maintainer for OK

**Comment by maxice8** [comment]
Please rebase.

@Gottox we will need repo cleanup since the new nodejs-lts version won't build on a few arches

**Comment by the-maldridge** [comment]
@maxice8 this breaks downstream users of yarn in several cases.  Most notably it breaks native builds for grafana.  If we're okay with breaking monitoring on arm then I guess we can go ahead, but I consider that a sub-optimal path.

**Comment by Cogitri** [comment]
Yup

**Pull request closed without merging**